tools/fbp.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/python3
```

Comment:
A long, long time ago...  There was a Unix system that used `#! /` (with the space) as the magic sequence to recognize shell scripts.  Every modern Unix of course uses just `#!`.

Reply:
Modern Unix systems do use `#!/` directives.

On a side note, this reminds me of using the following instead of the one in code:
```sh
#!/usr/bin/env python3
```

This makes it foolproof as the script can now be run by any `python3` binary located on path env variable

Reply:
Resolved in https://github.com/bcwhite-code/brians-blueprints/pull/21/commits/38a8c177499718492dc809063778110d0af04df0